.github/workflows/cmwl_pipeline_ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Run tests
         run: sbt coverage test coverage it:test
 
-      - name: Upload coverage data to Coveralls
-        run: sbt coverageAggregate coveralls
+      - name: Upload coverage data to Codecov
+        run: bash <(curl -s https://codecov.io/bash)
```

Comment:
There should be GitHub action for integration with `codecov`. Let's use it instead of raw bash command.